maintenance_releases.pt.Rmd

```diff
@@ -1,47 +1,47 @@
 ---
-aliases:
+aliases: 
   - releasing.html
 ---
 
-# Releasing a package {#releasing}
+# Liberação de um pacote {#releasing}
 
 ```{block, type="summaryblock"}
-Your package should have different versions over time: snapshots of a state of the package that you can release to CRAN for instance. These versions should be properly _numbered_, _released_ and _described in a NEWS file_. More details below.
+Seu pacote deve ter versões diferentes ao longo do tempo: instantâneos de um estado do pacote que você pode lançar no CRAN, por exemplo. Essas versões devem ser devidamente _numeradas_, _lançadas_ e _descritas em um arquivo NEWS_. Mais detalhes abaixo.
 
-Note that you could streamline the process of updating NEWS and versioning your package by using [the `fledge` package](https://github.com/cynkra/fledge).
+Observe que você pode simplificar o processo de atualização do NEWS e do controle de versão do seu pacote usando [o pacote `fledge`] (https://github.com/cynkra/fledge).
 ```
 
-## Versioning {#versioning}
+## Controle de versão {#versioning}
 
-- We strongly recommend that rOpenSci packages use semantic versioning. A detailed explanation is available in the [description chapter](https://r-pkgs.org/description.html#version).
+- Recomendamos enfaticamente que os pacotes rOpenSci usem controle de versão semântico. Uma explicação detalhada está disponível na seção [capítulo de descrição](https://r-pkgs.org/description.html#version).
```

Comment:
```suggestion
- Recomendamos enfaticamente que os pacotes rOpenSci usem controle de versão semântico. Uma explicação detalhada está disponível na seção [capítulo "lifecycle" do livro R packages](https://r-pkgs.org/lifecycle.html).
```

@pedropark99 this is a change we just made on main because the previous link was invalid. I used DeepL here too.

Reply:
Uhmm ok, I saw this weird output too, and we also adjusted the hyperlink in our review.